frontend/apply/routes.py

```diff
@@ -0,0 +1,13 @@
+from flask import render_template
+
+from pre_award.common.blueprints import Blueprint
+from pre_award.config import Config
+
+apply_bp = Blueprint("apply_routes", __name__, template_folder="templates")
+
+
+if Config.FLASK_ENV in ["dev", "development"]:
+
+    @apply_bp.route("/monolith")
+    def landing_page():
+        return render_template("apply/landing-mono.html.jinja")
```

Comment:
Ah sorry I wasn't clear :( I really really would like us to avoid any logic in our new "good" code that does checking like this on which environment we're running in to gate code/logic/features. I think we've had a few foot guns in the past with features that are meant to be developer-only ending up causing problems. This is unlikely to cause problems - of course - but I think I'd like it to be a red line for me 😓 

I think I mostly just meant: should we remove this altogether 🙏

Reply:
Remove this route

Reply:
Gone